net/netfilter/nf_tables_api.c

```diff
@@ -5281,6 +5294,13 @@ static int nft_set_elem_expr_setup(struct nft_ctx *ctx,
 	return -ENOMEM;
 }
 
+static void nft_setelem_remove(const struct net *net,
+			       const struct nft_set *set,
+			       const struct nft_set_elem *elem)
+{
+	set->ops->remove(net, set, elem);
+}
+
```

Comment:
This is technically introduced in 
`netfilter: nftables: add catch-all set element support` 
https://github.com/ctrliq/kernel-src-tree/pull/2/commits/628cf6169148cf5453ff84705ea254069b509428

The upstream is different but the code here matches the `4.18.0-534` we've based everything off of:
https://github.com/torvalds/linux/commit/aaa31047a6d25da0fa101da1ed544e1247949b40#diff-233a9dea2c513f5d208bba55e0f8ef4e8a29f5c584d30b5815cfa6c2cf5361ecR5635-R5644

Reply:
Yep, missed that one.  Will be fixed in the upcoming patch series.  The whole catchall piece of this has been a bit a challenge.  I've picked up some good learning though.